src/components/general/atomic/atomic.component.js

```diff
@@ -22,6 +22,9 @@ const loading = () =>
 const radioButton = () =>
   import(/* webpackChunkName: "radio-button" */ './radio-button/radio-button.component.vue')
 
+const ramlicomp = () =>
+  import(/* webpackChunkName: "ramlicomp" */ './ramlicomp/ramlicomp.component.vue')
```

Comment:
@mramli24  saya kira webpackChunkname jadi `general-component`

Reply:
okay, i will change it soon